llvm/cmake/modules/TableGen.cmake

```diff
@@ -23,18 +23,13 @@ function(tablegen project ofn)
 
   # Use depfile instead of globbing arbitrary *.td(s) for Ninja.
   if(CMAKE_GENERATOR MATCHES "Ninja")
```

Comment:
I suggest checking `CMP0116 NEW` here for external users who would set `OLD`. `OLD` may use fallback logic.